.gitattributes

```diff
@@ -1 +1,12 @@
+# LFS configuration for images from the wiki
 *.png filter=lfs diff=lfs merge=lfs -text
+
+# Exclude LFS-tracked files from the tarball
+/wiki/img/ export-ignore
```

Comment:
There are some .svg diagram sources there, is it ok to exclude them too?

Reply:
Source tarballs are meant to contain only files needed for installation. It's not uncommon for Linux distributions and others to track source tarball contents in git where spurious "stray" file from git like .gitattributes .gitignore or the .git directory can create confusion.
I haven't seen broken .svg files in the tarball so I don't think there's a need to exclude them, but right now it's not that important. Ideally in future it would be nice to have the full content of the wiki (with the "real" png data) as part of the release.

Reply:
What about .gitignore? I see it's there when I download ZIP. Should it be excluded too?

Reply:
Ideally yes.

Reply:
Ideally yes (but the focus of this PR was to remove the broken files). Want me to add the filter? Or you can edit the change.

Reply:
Please add yes